core/src/services/oss/core.rs

```diff
@@ -341,6 +301,48 @@ impl OssCore {
         Ok(req)
     }
 
+    pub async fn oss_append_object(
```

Comment:
This change seems not needed?

Reply:
Should we maintain the sequence of functions? For example, should we keep the `oss_xxx_request`  functions before the `oss_xxx` functions?

Reply:
Ideally, we should maintain the sequence while implementing changes. However, I am not interested in reviewing modifications that only alter the sequence.

Reply:
> Ideally, we should maintain the sequence while implementing changes. However, I am not interested in reviewing modifications that only alter the sequence.

I'm sorry for that, I will change the original function `oss_xxx_request` first